travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
+searched = {}
+
+
+def send(nick, dep, dest, date, r=None):
```

Comment:
@Adman the name of the function is extremely general. `send` can really mean anything.